### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by MicroBeaut**
@ArduinoBot
Changed the URL to be the same as the library name.

### CODE REVIEW

**Comment by MicroBeaut** [comment]
@ArduinoBot
A new release has been released for the URL change.

**Review by per1234 — COMMENTED** [review]
Hello @MicroBeaut. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]

**Comment by MatteoPologruto** [comment]
Done:
```
2023/05/09 13:51:35 Loaded 5829 libraries from DB
Changing URL of library RepeatButton from https://github.com/MicroBeaut/MicroBeaut-RepeatButton.git to https://github.com/MicroBeaut/RepeatButton.git
Success!
```

**Pull request merged**
